### PR TITLE
Switch to cozydb

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

**Comment by jsilvestre** [comment]
> +185  −514
Looks good so far :)

**Comment by clochix** [comment]
@frankrousseau could you please review this PR and merge it when it looks good ? Thanks !

File: package.json [hunk 1]
```diff
@@ -19,16 +19,15 @@
   "main": "server.js",
   "dependencies": {
     "americano": "0.3.7",
-    "americano-cozy": "0.2.6",
+    "cozydb": "0.0.2",
```
Comment (frankrousseau) [inline package.json]: Wrong version, the latest is 0.0.3.

File: server/helpers/file.coffee [hunk 2]
```diff
@@ -13,51 +11,6 @@ module.exports =
...
-        requester = downloader.download url, (stream) ->
-            if stream.statusCode is 200
-                stream.pipefilter = (source, dest) ->
-                    XSSmimeTypes = ['text/html', 'image/svg+xml']
-                    if source.headers['content-type'] in XSSmimeTypes
-                        dest.setHeader 'content-type', 'text/plain'
-
-                # Set headers from data system response data.
-                res.setHeader 'content-length', stream.headers['content-length']
-                res.setHeader 'content-type', stream.headers['content-type']
```
Comment (frankrousseau) [inline server/helpers/file.coffee]: Are you sure that CozyDB handles well these headers?

Comment (aenario) [inline server/helpers/file.coffee]: https://github.com/cozy/cozydb/blob/master/src/controller.coffee#L117-L127
It actually copy all headers from couchdb to the client

**Comment by aenario** [comment]
rebased against cozy/master, waiting for travis

**Comment by frankrousseau** [comment]
Thank you. Be ready for hotfix next week. I will try to upload some big files.

**Pull request merged**